vision.md

```diff
@@ -1,52 +1,41 @@
-
-### Future Directions
-
-Taking the above into account along with the requests we've received from users and our own interests, here are some directions we expect the next few releases of NUnit to take
-
-We'll continue to expand the available choices regarding **where tests are run**. We now support use of multiple AppDomains as well as separate processes. We expect to expand this to cover test execution under a particular subsystem or on a separate machine.
-
-**NUnit will be extensible.** We can't do everything for everybody but we will make it reasonably easy to extend NUnit. In many cases, users will be able to implement a plugin that provides a special feature outside of our scope by simply creating a new attribute that embeds the required logic. In more complex cases, particularly in extending runners, we will rely on a plugin-architecture.
+**NUnit is extensible.** We can't do everything for everybody but we will make it reasonably easy to extend NUnit. In many cases, users will be able to implement a plugin that provides a special feature outside of our scope by simply creating a new attribute that embeds the required logic. In more complex cases, particularly in extending runners, we will rely on a plugin-architecture.
 
 We'll continue to focus on **.NET development**. Testing non-.NET apps through tests written in a .NET language is an acceptable use of NUnit, but we won't give high priority to features that are needed only for that purpose.
 
-We'll remain **language neutral**. We'd like to work with **any** supported .NET languages. We currently support C#, VB.NET, C++, J# and F#. We expect to add Iron Python and Iron Ruby and will support other languages where people ask for it, particularly if they are able to pitch in and help.
+We'll remain **language neutral**. We'd like to work with **any** supported .NET languages. We currently support C#, VB.NET, C++ and F#. We will support other languages where people ask for it, particularly if they are able to pitch in and help.
 
-We'll remain **platform-neutral** and will **add more platforms**. When we use the term .NET, we don't only refer to MS.NET, but also to other ECMA-compliant implementations of the CLI. We currently support **MS.NET** and **Mono** and expect to add other platforms in the future. NUnit already allows running tests under different runtime platforms by use of a command-line option or by selecting an option in the Gui. We'll expand this to support running tests under the **compact framework** as well as **Silverlight**. We'll maintain separate download packages to support the different platforms, as we now do for Windows and Mono.
+We'll remain **platform-neutral** and will **add more CLI platforms** as needed. We currently support the **NET Framework**, **.NET Core** and **Mono**. NUnit already allows running tests under different runtime platforms by use of a command-line option or by selecting an option in the Gui.
```

Comment:
@CharliePoole Oh, that CLI! I read it as 'command line interface' and nothing else occurred to me. Maybe the sentence beside it about runners and command-line options did it. Perhaps it's only me then? I don't know, how about either 'platform' or if that's too open-ended, '.NET platform' which is something I hear it called?

I haven't brought this up yet but since it's in our vision doc, I'm 99% sure that `Gui` should always be `GUI`.